R/getCs2Profiles.R

```diff
@@ -0,0 +1,27 @@
+#' Get the List of cs2 Profiles
+#'
+#' A list of arguments to \code{\link{compareScenarios2}} is called a (cs2-) profile. This function loads a list of such profiles from a file. The default arguments are set such that the default profiles file of REMIND is loaded.
+#'
+#' @param path \code{character(1)}. Relative or absolute path to the profiles file.
+#' @param relativePath \code{character(1)}. Path to the profiles file relative to the REMIND directory, i.e., \code{remindDir}.
+#' @param remindDir \code{character(1)}. Path to the REMIND directory.
+#' @return A named list of profiles. The names are the names of the respective profile. Each profile is a named list. Those names correspond to arguments of \code{\link{compareScenarios2}}.
+#' @author Christof Schoetz
+#' @examples
+#' \dontrun{
+#' profiles <- getCs2Profiles() # load from default location
+#' profiles <- getCs2Profiles(remindDir = "path/to/remind")
+#' profiles <- getCs2Profiles(path = "path/to/profiles.json")
+#' profiles <- getCs2Profiles(relativePath = "path/relative/to/remindDir/profiles.json")
+#' }
+#' @export
+getCs2Profiles <- function(
+    path = file.path(remindDir, relativePath),
+    relativePath = "scripts/cs2/profiles.json",
+    remindDir = ".") {
```

Comment:
Kind of complicated with three arguments. If I specify for example `path` and `remindDir`, `remindDir` is simply ignored, right?
A simpler API would be to just have `path = "./scripts/cs2/profiles.json"` and let users of the function deal with defining the path themselves if they only know a `remindDir`.
Or, if only knowing the `remindDir` is common for users of the function, and we don't want to duplicate `scripts/cs2/profiles.json` in every user, maybe it would also be fine to have `path = "./scripts/cs2/profile.json"` by default and then allow the path to be a directory?
Something like
```
if (dir.exists(path)) {
    path = file.path(path, "scripts", "cs2", "profiles.json")
}
profiles <- jsonlite::read_json(path …)
```

Reply:
You are right. This is a bit overkill for a function that will mostly be called with its default argument... I will change it to your first suggestion